scripts/curate_data_axondeepseg_tem.py

```diff
@@ -0,0 +1,163 @@
+                        required=True,
+                        )
+    arguments = parser.parse_args()
+    return arguments
+
+
+def create_json_sidecar(path_folder_sub_id_bids, item_out):
+    data_json = {"PixelSize": [0.00236, 0.00236],
+                 "FieldOfView": [8.88, 5.39],
+                 "BodyPart": "BRAIN",
+                 "BodyPartDetails": "splenium",
+                 "SampleFixation": "2% paraformaldehyde, 2.5% glutaraldehyde",
+                 "Environment": "exvivo"
+                 }
+    item_out = item_out.replace('_TEM.png', '_TEM.json')
+    with open(os.path.join(path_folder_sub_id_bids, item_out), 'w') as json_file:
+        json.dump(data_json, json_file, indent=4)
+
+
+def main(root_data, output_data):
+    # Remove macOS .DS_Store
+    subprocess.run(["find", root_data, "-name", ".DS_Store", "-type", "f", "-delete"], check=True)
+
+    # Remove output_data if exists to start clean
+    if os.path.isdir(output_data):
+        shutil.rmtree(output_data)
+    os.makedirs(output_data, exist_ok=True)
+
+    contents_ds = [subdir for subdir in os.listdir(root_data) if os.path.isdir(os.path.join(root_data, subdir))]
+    contents_ds.sort()
+
+    # Loop across contents of each subdirectory
+    for subdir in contents_ds:
+        sub_id = "sub-nyuMouse" + subdir.split('_')[3]
+        sample_id = subdir.split('_')[4]
+        path_subdir = os.path.join(root_data, subdir)
+        contents_subdir = os.listdir(path_subdir)
+        sub_bids_full = sub_id + "_sample-" + sample_id
+        for file in contents_subdir:
+            path_file_in = os.path.join(path_subdir, file)
+            if file in images:
+                path_sub_id_dir_out = os.path.join(output_data, sub_id, 'microscopy')
+                path_file_out = os.path.join(path_sub_id_dir_out, sub_bids_full + images[file])
+                os.makedirs(path_sub_id_dir_out, exist_ok=True)
+                create_json_sidecar(path_sub_id_dir_out, sub_id + '_TEM.png')
+            if file in der:
+                path_sub_id_dir_out = os.path.join(output_data, 'derivatives', 'labels', sub_id, 'microscopy')
+                path_file_out = os.path.join(path_sub_id_dir_out, sub_bids_full + der[file])
+                os.makedirs(path_sub_id_dir_out, exist_ok=True)
+            shutil.copy(path_file_in, path_file_out)
+
+    sub_list = os.listdir(output_data)
+    sub_list.remove('derivatives')
+    sub_list.sort()
+
+    # Create participants.tsv and samples.tsv
+    with open(output_data + '/samples.tsv', 'w') as samples, \
+            open(output_data + '/participants.tsv', 'w') as participants:
+        tsv_writer_samples = csv.writer(samples, delimiter='\t', lineterminator='\n')
+        tsv_writer_samples.writerow(["participant_id"])
+        tsv_writer_participants = csv.writer(participants, delimiter='\t', lineterminator='\n')
+        tsv_writer_participants.writerow(["participant_id", "sample_id", "sample_type"])
```

Comment:
Columns order should be: `sample_id`, `participant_id`, `sample_type`

Reply:
done https://github.com/neuropoly/data-management/pull/103/commits/32a30d1ce4c4d6d8f4d0505f86e40695d8274e98